### ISSUE TITLE
Lost keyboard and mouse after installation

### ISSUE THREAD

**Issue by Gabotero** [JaKooLit/Debian-Hyprland#96]
Hi!

- I just went through the installation for kali linux following this tutorial: https://www.youtube.com/watch?v=CYHj9OCmujc
- I downloaded and installed the latest kali linux (June 1, 2024) in ssd, without any desktop environment, same as in the video.
- I also removed/blacklisted nouveau drivers to let the installer deal with the nvidia drivers (at least I understood that the installation script would install them for me).
- Everything looks fine, I reboot and get the same login screen as in the video.

- However, when the login screen pops up, I cannot use mouse or keyboard at all. Not even to press ctrl+alt+F2/F3 to change to text mode. On the other hand, I am able to use keyboard if I boot into text mode from the grub menu.

Do you have an idea of what could be happening or in what direction I could keep researching?

I include a tar file with the logs. As far as I know, de SDDM installation was successful 
https://drive.google.com/file/d/1ivGviWrD4ENNOcBgR-M_UdrvPXeGk6oB/view?usp=sharing
 

Thank you in advance for any help.

Best

**Comment by Gabotero** [comment]
Quick update:

- I think it was a problem related with the installation of the nvidia drivers.
- I followed this alternative tutorial where they install kali linux with xfce first: https://www.youtube.com/watch?v=NtpRtSBjz3I
- This enabled me to better troubleshoot any potential problem (since with the previous method, I wasn't able to test the nvidia drivers with xfce)
- Later, I tried to install the nvidia drivers. It seemed good again, but the "nvidia-smi" command was not showing the appropriate output. I had to google a bit to find the solution: https://forums.developer.nvidia.com/t/nvidia-smi-communication-error-in-kali-linux-in-kernal-6-5-0-kali3-amd64/280674
- Basically, it seems an extra step (install kernel headers) you need to follow if you own a laptop with the RTX 40XX Max-Q / Mobile series.
- Then, I just continued with the video tutorial and everything went smooth.

Thanks and I hope it helps somebody.
